src/main/java/mods/railcraft/common/carts/EntityLocomotive.java

```diff
@@ -463,11 +463,14 @@ private boolean cartVelocityIsGreaterThan(float vel) {
     }
 
     public int getDamageToRoadKill(EntityLivingBase entity) {
-        if (entity instanceof EntityPlayer) if (ItemOveralls.isPlayerWearing((EntityPlayer) entity)) {
+        if (entity instanceof EntityPlayer) {
             ItemStack pants = ((EntityPlayer) entity).getCurrentArmor(MiscTools.ArmorSlots.LEGS.ordinal());
```

Comment:
Since you're touching this code, change the if to `if (entity instanceof EntityPlayer player)` to avoid an unnecessary cast.

Reply:
Similarly the pants variable can be renamed to `stack` until we know they're pants and you can do `instanceof ISafetyPants pants` below.